README.md

```diff
@@ -33,16 +33,34 @@ export default defineConfig(() => {
 
 `src/routes/service-worker.ts`:
 
-```ts
+```diff
 import { setupServiceWorker } from "@builder.io/qwik-city/service-worker";
 import { setupPwa } from "@qwikdev/pwa/sw";
 
 setupServiceWorker();
-setupPwa();
 
-addEventListener("install", () => self.skipWaiting());
++setupPwa();
+
+- addEventListener("install", () => self.skipWaiting());
+
+- addEventListener("activate", () => self.clients.claim());
+
+- declare const self: ServiceWorkerGlobalScope;
+```
+
+By default, your application will be auto-updated when there's a new version of the service worker available and it is installed: in a future version, you will be able to customize this behavior to use `prompt` for update:
+```ts
+import { setupServiceWorker } from "@builder.io/qwik-city/service-worker";
+import { setupPwa } from "@qwikdev/pwa/sw";
+
+setupServiceWorker();
+setupPwa("prompt");
+```
 
-addEventListener("activate", () => self.clients.claim());
+`public/manifest.json`:
+```diff
+"background_color": "#fff",
++ "theme_color": "#fff",
```

Comment:
could you mention in docs why this is needed?

Reply:
I need to check the errors, the theme_color is a minimal requirement, I'll add a reference to the document: https://vite-pwa-org.netlify.app/guide/pwa-minimal-requirements.html

Reply:
a simple sentence with a reference can be enough.

Reply:
done